src/test/java/domain/CarsTest.java

```diff
@@ -33,42 +33,20 @@ void participantTest(String name1, String name2, String name3) {
     }
 
     @Test
-    @DisplayName("우승자의 위치를 숫자로 반환한다.")
-    void getWinnerLocation() {
+    @DisplayName("각 자동차를 조건에 맞춰 이동한다")
+    void moveCarsTest() {
         //given
-        Car one = new Car("one");
-        Car two = new Car("two");
-        Car three = new Car("three");
+        Car one = new Car("one", 0);
+        Car two = new Car("two", 0);
+        Car three = new Car("three", 0);
         Cars cars = new Cars(Arrays.asList(one, two, three));
 
         //when
-        three.move(4);
-        three.move(4);
-        three.move(4);
+        cars.moveCars(new FixedNumberGenerator());
```

Comment:
```suggestion
        cars.moveCars(() -> 4);
```
만들어주신 `NumberGenerator`는 `@FunctionalInterface`이기 때문에 람다식으로도 표현이 가능합니다.